src/app/view/components/_common/site-selector/site-selector.component.ts

```diff
@@ -50,6 +50,7 @@ export class SiteSelectorComponent implements OnInit, OnChanges {
     currentSite: Observable<Site>;
     totalRecords: number;
     sitesCurrentPage: Site[];
+    sites: any[];
```

Comment:
Should this be ```Site[]````?

Reply:
the old interface for site only has 3 attributes, in order to use it with the new fields from the backend that interface needs to be updated with all the attributes which are around 20

Reply:
I think we can use Site anyway, the miss attributes just will be ignore

Reply:
I did that test before and since the interface doesn't has all the attributes, when paginationService.getCurrentPage() returns the objects, those objects contains more attributes that don't match the interface and in the end, sites[] variable endup with length 0